### ISSUE TITLE
Results on MPII test dataset

### ISSUE THREAD

**Issue by zhangrj91** [ilovepose/DarkPose#32]
Hi, thanks for your great work!

We would cite your paper, could you give me the result of your model on MPII test dataset?

**Comment by xizero00** [comment]
Hi,
We do not submit the results of the MPII test set to the author of MPII dataset, so we don't have the performance on the MPII test set.
You can use the pre-trained model to generated the results on the MPII test set and submit them to the author of the MPII dataset to obtain the performance.
